src/Tools/NodeDocumentationMarkdownGenerator/Commands/RenameCommand.cs

```diff
@@ -61,7 +61,7 @@ private static void RenameFile(string file)
 
         private static void RenameFile(string file, string baseName, string shortName)
         {
-            var content = File.ReadAllText(file);
+            var content = HttpUtility.UrlDecode(File.ReadAllText(file));
```

Comment:
why have you only modified this overload of `RenameFile` - now the others are inconsistent? Is there a good reason?

Reply:
The problem is only happening when is trying to replace the image name inside the .md file with the hashed file name and this RenameFile(string file, string baseName, string shortName) method is doing that task, 

e.g. for ByOriginVectors node the next values will be replaced.
baseName = "Autodesk.DesignScript.Geometry.CoordinateSystem.ByOriginVectors(origin, xAxis, yAxis, zAxis)" 
shortName = "KLFVJ5DI7APVNODARMSE3JM5C27DG62OXUUQRS3HCDTZYO6IBOZA"

The other RenameFile(string file) function is calling RenameFile(string file, string baseName, string shortName), then I don't understand what you mean by "inconsistent in behavior".

Reply:
Thanks for the explanation- that makes sense, so now all replace methods will eventually call down to this one and have the same behavior with decoding?

Reply:
Sorry, typo, I meant `Rename` not replace.
ReplaceFile
ReplaceDirectory

Reply:
yes, it will do the decoding whenever replaces the content inside a **md** file.
Here you can see a flow diagram showing that when using "rename" option in both cases will be decoding, I hope this diagram shows the behavior in a more clear way:
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/c359f2e0-d185-4fe3-a0fe-9134915b777c)